packages/@ember/octane-app-blueprint/index.js

```diff
@@ -14,8 +14,9 @@ module.exports = {
     return Promise.all([
       latestVersion('ember-cli', {version: 'beta'}),
       getRepoVersion('ember-cli', 'ember-cli-htmlbars', 'colocation'),
+      latestVersion('ember-data', {version: 'beta'}),
       latestVersion('ember-source', {version: 'beta'})
-    ]).then(([emberCLI, emberCLIHTMLBars, emberSource]) => {
+    ]).then(([emberCLI, emberData, emberCLIHTMLBars, emberSource]) => {
       let name = stringUtil.dasherize(options.entity.name);
```

Comment:
Ember data should be the 3rd entry in here?

Reply:
ooopss